includes/user.php

```diff
@@ -232,6 +247,27 @@ public static function set_introduction_viewed( array $data ) {
 		update_user_meta( get_current_user_id(), self::INTRODUCTION_KEY, $user_introduction_meta );
 	}
 
+	public static function register_as_beta_tester( array $data ) {
+
+		add_filter( 'https_ssl_verify', '__return_false' );
```

Comment:
Remove.

Reply:
for sure? this exists in all other elementor-api calls